caffe-grid/pom.xml

```diff
@@ -18,11 +18,28 @@
     <maven.compiler.target>1.7</maven.compiler.target>
     <hadoop.version>2.5.0</hadoop.version>
     <scala.version>2.10.4</scala.version>
```

Comment:
To avoid duplicated info, let's change it to define scala.major.version based on scala.version. Say, 

<scala.major.version>2.11</scala.major.version>
<scala.version>${scala.major.version}.7</scala.version>